### PR TITLE
Improve sub headline

### PR DESCRIPTION
**Pull request by vog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Berlin Hack and Tell
 
-[Meetup](https://www.meetup.com/de-DE/Berlin-Hack-and-Tell)
+Last Tuesday on Jan-Nov | [Meetup Page](https://www.meetup.com/de-DE/Berlin-Hack-and-Tell) | [@c-base](https://www.c-base.org)
```
Comment (ligi) [inline README.md]: I would like Jan..Nov more :-) But then we also really have to make sure there is no more breaks in between - there have been times without BHNT but it was Jan..Nov - cannot compute

**Comment by vog** [comment]
@ligi What about:

> Last Tuesday each month | ...

That is simpler and at least obviously not-quite-correct.

**Comment by ligi** [comment]
that's fake news!-)
what about "Last Tuesday each(~) month"
or eachish

**Comment by vog** [comment]
I prefer each(~) of eachish. Adjusted patch. Please review again.

**Comment by ligi** [comment]
:+1:

**Pull request merged**